airflow/providers/amazon/aws/operators/batch.py

```diff
@@ -126,6 +134,7 @@ def __init__(
         aws_conn_id: Optional[str] = None,
         region_name: Optional[str] = None,
         tags: Optional[dict] = None,
+        wait_for_completion: bool = True,
```

Comment:
Hi, I know that is just draft PR.
But would you like also add changes in the extra link conditional property

https://github.com/apache/airflow/blob/7d8a17b2107a962e1ffcdb22fc13e1d23a98fdb7/airflow/providers/amazon/aws/operators/batch.py#L104-L111

If `wait_for_completion` sets to `False` then there is no reason add `BatchJobDefinitionLink()` and  `BatchJobQueueLink()` since it executed at `monitor_job` method